Add output how-tos (#465)
iotaledger/iota-sdk: documentation/sdk/docs/how_tos/outputs/output_unlock_conditions.mdx

---
title: Output Unlock Conditions

> description: 'How to create ouputs with different unlock conditions'

Reviewer: Why do I think about "How to create Aua" when reading this? :trollface:
Joke, but I think it generally doesn't sound that great, how about this? Or does it even need to have "How to"?
```suggestion
description: 'How to set unlock conditions for outputs'
```

import PythonCode from '!!raw-loader!../../../../../bindings/python/examples/how_tos/outputs/output_unlock_conditions.py';

Outputs can have different unlock conditions. This how to exaplains how to add them to an output.

:::info Output Unlock Conditions

Different outputs can have different unlock conditions. This tables shows you which ouput is compatible with which unlock condition. You can find the output specs in [TIP-018](https://wiki.iota.org/tips/tips/TIP-0018)

| Output   | Address | Storage Deposit Return | Timelock | Expiration | State Controller Address | Governor Address | Immutable Alias Address |
| -------- | :-----: | :--------------------: | :------: | :--------: | :----------------------: | :--------------: | :---------------------: |
| Basic    | x       | x                      | x        | x          |                          |                  |                         |
| Alias    |         |                        |          |            | x                        | x                |                         |
| Foundry  |         |                        |          |            |                          |                  | x                       |

> | NFT      | x       | x                      | x        | x          |                          |                  |                         |

Reviewer 2: Love that

:::

## Code Example

The following example will:

> 1. Create multiple outputs with all possible features

Reviewer: ```suggestion
1. Create multiple outputs with all possible unlock conditions
```
